webservices/resources/legal.py

```diff
@@ -103,6 +103,9 @@ def get(self, q, from_hit=0, hits_returned=20, type='all', **kwargs):
                 .extra(size=hits_returned, from_=from_hit) \
                 .index('docs')
 
+            if type == 'advisory_opinions':
```

Comment:
I don't think it should hold this PR up, but maybe there should be a flag to over-ride the logic and search all the AOs? That would build in some flexibility for the future.